examples/saturation_test/md5_thread.c

```diff
@@ -197,7 +197,7 @@ void *MB_THREAD_FUNC(void *arg)
 
       out:
 	free(ctxpool);
-	free(mgr);
+	aligned_free(mgr);
```

Comment:
Could you explain why this change is needed? From what I see in `man` use of `free()` is legitimate.
```
The  function  posix_memalign()  allocates  size bytes and places the address of the allocated memory in *memptr.  The address of the allocated memory will be a multiple of alignment, which must be a
power of two and a multiple of sizeof(void *). 
This address can later be successfully passed to free(3).
```

Reply:
Sorry, I missed the point it is for Windows. There are more changes needed here to make it compile on Windows, I am afraid. Only SHA1 and GCM examples compile on Windows.